java/feature-layer-feature-service/src/main/java/com/esri/arcgisruntime/samples/featurelayerfeatureservice/MainActivity.java

```diff
@@ -27,73 +25,54 @@
 import com.esri.arcgisruntime.mapping.Viewpoint;
 import com.esri.arcgisruntime.mapping.view.MapView;
 
-
 public class MainActivity extends AppCompatActivity {
 
-    private MapView mMapView;
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-
-        // inflate MapView from layout
-        mMapView = (MapView) findViewById(R.id.mapView);
-
-        // create a map with the terrain with labels basemap
-        ArcGISMap map = new ArcGISMap(Basemap.createTerrainWithLabels());
-        //set an initial viewpointf
-        map.setInitialViewpoint(new Viewpoint(new Point(-13176752, 4090404, SpatialReferences.getWebMercator()), 500000));
-
+  private MapView mMapView;
 
-        // create feature layer with its service feature table
-        // create the service feature table
-        ServiceFeatureTable serviceFeatureTable = new ServiceFeatureTable(getResources().getString(R.string.sample_service_url));
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
 
-        // create the feature layer using the service feature table
-        FeatureLayer featureLayer = new FeatureLayer(serviceFeatureTable);
+    // inflate MapView from layout
+    mMapView = (MapView) findViewById(R.id.mapView);
 
-        // add the layer to the map
-        map.getOperationalLayers().add(featureLayer);
+    // create a map with the terrain with labels basemap
+    ArcGISMap map = new ArcGISMap(Basemap.createTerrainWithLabels());
+    //set an initial viewpointf
+    map.setInitialViewpoint(new Viewpoint(new Point(-13176752, 4090404, SpatialReferences.getWebMercator()), 500000));
 
-        // set the map to be displayed in the mapview
-        mMapView.setMap(map);
+    // create feature layer with its service feature table
+    // create the service feature table
+    ServiceFeatureTable serviceFeatureTable = new ServiceFeatureTable(
+        getResources().getString(R.string.sample_service_url));
 
-    }
+    // create the feature layer using the service feature table
+    FeatureLayer featureLayer = new FeatureLayer(serviceFeatureTable);
 
-    @Override
-    public boolean onCreateOptionsMenu(Menu menu) {
-        // Inflate the menu; this adds items to the action bar if it is present.
-        getMenuInflater().inflate(R.menu.menu_main, menu);
-        return true;
-    }
+    // add the layer to the map
+    map.getOperationalLayers().add(featureLayer);
 
-    @Override
-    public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
Getting rid of the options menu is a good idea as it's unused - but should delete the menu_main resource file too in this case.

Reply:
done